spec/models/customer_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper' 
+
+RSpec.describe Customer, type: :model do 
+  describe "relationships" do 
+    it { should have_many :items }
```

Comment:
Customers have many Items.
Items have many Customers.

So here will will need to set up our many-to-many associations
```ruby
 it { should have_many(:customer_items) }
 it { should have_many(:items).through(:customer_items) }
```